nemo/collections/asr/models/ctc_bpe_models.py

```diff
@@ -67,22 +67,38 @@ def __init__(self, cfg: DictConfig, trainer=None):
         vocabulary = self.tokenizer.tokenizer.get_vocab()
 
         # Set the new vocabulary
-        cfg.decoder.params.vocabulary = ListConfig(list(vocabulary.values()))
+        if "params" in cfg.decoder:
+            cfg.decoder.params.vocabulary = ListConfig(list(vocabulary.values()))
+        else:
+            cfg.decoder.vocabulary = ListConfig(list(vocabulary.values()))
```

Comment:
Does this work without OmegaConf's open_dict() or is this not an OmegaConf config?

Reply:
I think it is an OmegaConf but we can add stuff by default without open_dict(). Don't we need to call OmegaConf.set_struct() to enable such restrictions?

Reply:
Please wrap open_dict() to be safe

Reply:
For this specific part, we assumed that vocabulary is mandatory and always in the config file. Do we still need open_dict for here?

Reply:
Apparently, OmegaConf's default is not to use set the config dictionary to fixed structure. But I feel like this default has changed from prior versions. So to be safe, I think we should add open_dict for now.
Also, resolving this point.